pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -597,15 +614,22 @@ func (dm *DockerManager) runContainer(
 		cpuShares = milliCPUToShares(cpuRequest.MilliValue())
 	}
 	var devices []dockercontainer.DeviceMapping
+	var nvidiaGPUPaths []string
+	nvidiaGPUPlugin := dm.GetNvidiaGPUPlugin()
 	if nvidiaGPULimit.Value() != 0 {
```

Comment:
I think for GPU just need request is OK, I'll change it and add the checking code in canAdmitPod.